.circleci/config.yml

```diff
@@ -24,8 +29,9 @@ jobs:
       - run: cat packages/*/package.json | shasum > .dependencies_checksum
       - restore_cache:
           key: dependency-cache-{{ checksum ".dependencies_checksum" }}
+      - attach_workspace:
+          at: /home/circleci/project
       - run: yarn # symlink packages' node_modules
```

Comment:
```suggestion
```

Reply:
I put that there to make it explicitly clear why `yarn` is being run

Without context you'd be tempted to remove it because you'd think `yarn` has already ran to pull the dependencies so it doesn't need to be run again.

Reply:
I think we won't need it though because the workspace is persisted, thus the symlinks are too

Reply:
In this PR i persist only the dist/ and build/ folders if that matters.

Reply:
Cool - if it works with the updated config then it's all good

This comment was just for _that_ config which seems stale now

Reply:
Hm ok let's explore this later then, this PR looks good